doc/whats-new.rst

```diff
@@ -100,6 +100,8 @@ Bug fixes
   (:issue:`3402`). By `Deepak Cherian <https://github.com/dcherian/>`_
 - Allow appending datetime and bool data variables to zarr stores.
   (:issue:`3480`). By `Akihiro Matsukawa <https://github.com/amatsukawa/>`_.
+- Make :py:func:`~xarray.concat` more robust when concatenating variables present in some datasets but
+  not others (:issue:`508`). By `Scott Chamberlin <http://github.com/scottcha>`_.
```

Comment:
```suggestion
  not others (:issue:`508`). By `Scott Chamberlin <https://github.com/scottcha>`_.
```
until we merge #3516, the some of the old entries don't have this format, but let's have all new entries match it.